guibot/region.py

```diff
@@ -608,11 +672,13 @@ def hover(self, target_or_location: "Match | Location | str | Target") -> "Match
 
         return match
 
-    def click(self, target_or_location: "Match | Location | str | Target",
-              modifiers: list[str] = None) -> "Match | None":
+    def click(
+        self,
+        target_or_location: "Match | Location | str | Target",
+        modifiers: list[str] = None,
+    ) -> "Match | None":
         """
-        Click on a target or location using the left mouse button and
-        optionally holding special keys.
+        Click on a target or location using the left mouse button and special keys.
```

Comment:
Click on a target or location using the left mouse button.

Optionally we can hold some special keys.

Reply:
Done

Reply:
Done